cmd/otelcol/main.go

```diff
@@ -25,14 +26,19 @@ import (
 )
 
 func main() {
-	handleErr := func(message string, err error) {
-		if err != nil {
-			log.Fatalf("%s: %v", message, err)
-		}
+	params, err := applicationParameters()
+	if err != nil {
+		log.Fatal(err)
 	}
 
+	run(params)
+}
+
+func applicationParameters() (service.Parameters, error) {
```

Comment:
We use "service" and "application" terms interchangeably in naming and errors, should probably settle on one.

Reply:
I actually got rid of this method in favour of leaving the main function closer to how it was before, but yea the naming discrepancy between service & application is a little confusing, especially with the added concept of now having a "Windows _Service_" wrapper around the service/application. If you think I could improve any other naming in this PR, lmk